subgraphs/stakedao/src/Oracle/index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Address,
+  BigDecimal,
+  BigInt,
+  dataSource,
+  log,
+} from "@graphprotocol/graph-ts";
+
+import * as constants from "./common/constants";
+import { getCurvePriceUsdc } from "./routers/CurveRouter";
+import { getPriceUsdc as getPriceUsdcUniswap } from "./routers/UniswapRouter";
+import { getPriceUsdc as getPriceUsdcSushiswap } from "./routers/SushiSwapRouter";
+
+export function getUsdPricePerToken(tokenAddr: Address): BigDecimal[] {

```

Comment:
I think you can do this `[BigDecimal, BigDecimal]` as return value. Make a comment on what they are.

Reply:
Generally, a multiple return type throws an error in `thGraph`, [here](https://discord.com/channels/438038660412342282/438070183794573313/963050151390367824) is my question on theGraph Discord.